hw/ip/otbn/dv/otbnsim/sim/insn.py

```diff
@@ -218,7 +218,7 @@ def execute(self, state: OTBNState) -> None:
         val1 = state.gprs.get_reg(self.grs1).read_unsigned()
         val2 = state.gprs.get_reg(self.grs2).read_unsigned()
         if val1 == val2:
-            state.pc_next = self.offset
+            state.set_pc_next(self.offset)
```

Comment:
Note: We have a couple options to implement this behavior. If our goal is to make the code use `state.pc_next` instead of an explicit setter we can use a property setter (this code shows up in the docs). But then we have other set_* functions in the OTBNState class currently, so we would loose a bit of consistency there. I'm fine either way.

Reply:
I'm happy to use a setter, but can we call it `set_next_pc`? I think that might be a bit clearer in the documentation (where readers won't know anything about a variable called "pc_next")